test/Directory.Build.props

```diff
@@ -2,7 +2,7 @@
   <Import Project="..\Directory.Build.props" />
 
   <PropertyGroup>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFrameworks>net5.0;net48</TargetFrameworks>
```

Comment:
Don't you want to condition that net48 only gets added in Windows? Is this Directory.Build.props file used by any other project apart from the two projects being modified below? If the answer is no, then you can probably consider removing TargetFrameworks from this file and instead have it be defined in the projects directly